torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -419,6 +419,27 @@ def pad_bounding_box(
 crop_image_pil = _FP.crop
 
 
+def crop_bounding_box(
+    bounding_box: torch.Tensor,
+    format: features.BoundingBoxFormat,
+    top: int,
+    left: int,
+) -> torch.Tensor:
+    shape = bounding_box.shape
+
+    bounding_box = convert_bounding_box_format(
+        bounding_box, old_format=format, new_format=features.BoundingBoxFormat.XYXY
+    ).view(-1, 4)
+
+    # Crop or implicit pad if left and/or top have negative values:
+    bounding_box[:, 0::2] -= left
```

Comment:
For batch support it may be better to instead use `bounding_box[..., 0::2]`...

Reply:
Good catch. I was under the impression `...` was not supported by `torch.jit.script` in general, but this seems fine:

```py
@torch.jit.script
def foo(x: torch.Tensor) -> torch.Tensor:
    y = x.clone()
    y[..., 0::2] += 1
    return y
```

It seems, only the explicit indices are not supported:

```py
@torch.jit.script
def bar(x: torch.Tensor) -> torch.Tensor:
    y = x.clone()
    y[..., [0, 2]] += 1
    return y
```

```
RuntimeError: 
Ellipses followed by tensor indexing is currently not supported:
[...]
def bar(x: torch.Tensor) -> torch.Tensor:
    y = x.clone()
    y[..., [0, 2]] += 1
    ~~~~~~~~~~~~~ <--- HERE
    return y
```